Automattic/newspack-theme
# Add RTL styles for style packs

### All Submissions:


### Changes proposed in this Pull Request:

Rather than layering RTL styles on top of the existing style.css, the Newspack theme follow's Twenty Nineteen's approach and generates a completely separate stylesheet (which is much easier for managing changes and making sure things aren't missed!).

This PR adds a line in the build process for generating an RTL stylesheet for Style 1, and adds the initial styles.

Closes #48.

### How to test the changes in this Pull Request:

1. Apply the PR.
2. Navigate to Customizer > Style Pack, and switch to Style 1.
3. View the RTL version of the styles. The easiest way to do this is installing and activating RTL Tester (https://wordpress.org/plugins/rtl-tester/), and click 'Switch to RTL' in the admin toolbar.
4. Confirm that the site still looks like style pack 1 (light grey background, all fonts serif). 

### Other information:
